test/compose/cdi_device/setup.sh

```diff
@@ -0,0 +1,9 @@
+if is_rootless; then
+    reason=" - can't write to /etc/cdi"
```

Comment:
Would using `/var/run/cdi` allow for these to run in rootless environments?

Reply:
AFAICS, we are not currently using that in Podman (probably we should).

Reply:
Yes but even when we add that a rootless user would still have no write privs there.
Someone should finish https://github.com/containers/podman/pull/21448 so the test can pick their own private dir for the configs.

Reply:
I can't remember if the handling of `/etc/cdi` and `/var/run/cdi` respect the relevant `XDG_` envvars.